### DISCUSSION TITLE
Is there a way to get Video Stream of one Camera to Another?

### DISCUSSION THREAD

**Discussion by ThenoobMario** [AxisCommunications/acap-native-sdk-examples#170] [Q&A] [answered]
Hello,

I was wondering if there was a way to get the video stream of one IP Camera to another IP Camera (Connected to the same local network). If yes, how can I do that?

**Comment by pataxis** [comment]
Hi @ThenoobMario , thank you for your question!

You could have a look at the VAPIX APIs for [Video Streaming](https://www.axis.com/vapix-library/subjects/t10175981/section/t10051110/display).

Out of curiosity, what's your use case?

**Reply by ThenoobMario** [reply]
Hello @pataxis,

Thanks for the reply. I have gone through the video streaming API and figured out the URL structure to be sent.

My use case involves getting the video streams from two IP cameras to a third one and then sending it forward to a VMS. These three streams need to be put/stitched together in one newer stream. 

It would be great if you could share any ideas for this. 

Thank you.

**Reply by rocky-ji** [reply, marked as answer]
You can get rtsp steam from other camera on one camera, but I am not too sure whether the camera have enough resource to decode all the video and stitche fusion and encode them.

not too sure about which kind of video you want to get, maybe a 180 degree panoramic is a good choice, it can stream a 180 degree rtsp video directly.https://www.axis.com/products/multisensor-cameras

but if your cameras are installed in different position, you will have to do this by yourself, most time server is needed since you may also need do some video brightness/saturation adjust
